model/image.go

```diff
@@ -107,9 +107,9 @@ func NewSoftwareImage(
 	return &SoftwareImage{
 		SoftwareImageMetaConstructor:         *metaConstructor,
 		SoftwareImageMetaArtifactConstructor: *metaArtifactConstructor,
-		Modified: &now,
-		Id:       id,
-		Size:     artifactSize,
+		Modified:                             &now,
```

Comment:
Is this formatted correctly?

Reply:
according to my go fmt, it is NOW formatted correctly, it was wrong before, no idea why.